docs/Conformance Criteria/README-Conformance Criteria V2.md

```diff
@@ -0,0 +1,358 @@
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">6</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Using .zowe in user settings:</b> Extender avoids starting setting names with the prefix `zowe.`, which is reserved for Zowe Explorer and and other extensions maintained by the Zowe Project
+.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">7</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>Error message consistency:</b> Extension follows the recommended error message format indicated in the <a href=https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/extenders/Error%20Handling.md>Zowe Explorer extensibility documentation</a> to provide a consistent user experience with Zowe Explorer.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">8</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>Zowe SDK usage:</b> Extension utilizes the available Zowe SDKs that standardize z/OS interactions as well as other common capabilities that are used by many other Zowe extensions and tools unless the extension's goal is to provide a new implementation with clearly stated goals.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">9</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Sharing of profiles with Zowe CLI:</b> Extensions that utilize Zowe CLI profiles must share the created profile instances between Zowe CLI and the Zowe Explorer extension that utilize them.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">10</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>v1 Profile Support:</b>Extension has a backwards compatibility and it is able to support v1 type of profiles.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555" rowspan=5>11</th>
+   <th style="background-color:#555555"></th>
+   <th style="background-color:#AAAAAA" colspan=2>Mark (a) or (b) or (c)</th>
+   <th style="background-color:#AAAAAA"></th>
+   <td style="text-align:center">Extension uses the extensibility APIs provided by Zowe Explorer. Supported methods include:<p style="color:red">(Please select all that apply _a_, _b_, or _c_)</td>
```

Comment:
We could have an extension applying for conformance that doesn't fall into one of these categories. I think we should explicitly mention that. I also think "Extension Accessing Profiles" is too broad. My understanding is that this section only applies to VS Code extensions that want to access the **same** Zowe CLI profiles that Zowe Explorer uses. Therefore a plug-in accessing other profiles doesn't apply. They would just need to satisfy the "Sharing of profiles with Zowe CLI" requirement.

Reply:
Based on our discussion today, perhaps this should be adjusted to "Extension Leveraging Zowe Explorer Profiles Cache". This would be a requirement for extensions using the shared z/OSMF profile. However, it would just be a best practice for extensions leveraging other profiles. The feedback from some extenders was that it should not be strictly required to avoid a dependency on Zowe Explorer as a separate extension.

Reply:
The Zowe Explorer APIs were created specifically with extenders in mind and the `required` status was set so all profile handling, error handling (new and a best practice at this time), and shared code for keytar to make it easier on extenders as well as have all of these things handled in the same way.  The APIs have been in place for a year now and the FTP VSC ext, CICS VSC ext, and some external VSC extensions have already adopted the use of APIs. We had this as a `best practice` in V1.  Does the CLI `require` the dependency of the SDKs? 
This decision may need to be discussed some more, we don't want to make it harder on extenders but like @phaumer said "If they don't fall into any of these categories, would they be consider an extender?"  It would be good to have some examples from these "extenders" on how they are actually extending Zowe Explorer.  With this information we may be able to add a new category that they would fall into.